CardBox/CardGameEngine/Card.swift

```diff
@@ -56,7 +65,17 @@ class Card: Identifiable {
         }
     }
 
+    func canPlay(by player: Player, gameRunner: GameRunnerReadOnly, on target: GameplayTarget) -> Bool {
```

Comment:
Same here, I think the logic of canPlay should be on the players instead. The card does not hold any canPlay information

Reply:
Done!